app/src/main/java/net/gsantner/markor/model/Constants.java

```diff
@@ -18,8 +18,18 @@
     public static final String[] KEYBOARD_SHORTCUTS_MORE = {"!", "+", "~", ":", "<", ">"};
     public static final String UTF_CHARSET = "utf-8";
 
-    public static final String MD_EXT = ".md";
-
+    public static final String MD_EXT1 = ".md";
+    public static final String MD_EXT2 = ".markdown";
+    public static final String MD_EXT3 = ".mkd";
+    public static final String MD_EXT4 = ".mdown";
+    public static final String MD_EXT5 = ".mkdn";
+    public static final String MD_EXT6 = ".pdf";
```

Comment:
markdown cannot be saved as picture or pdf. only be exported, so these should not be listed

Reply:
Oh that just  got out of my mind.